exporters/otlp-metrics/build.gradle

```diff
@@ -0,0 +1,27 @@
+plugins {
+    id "java"
+    id "maven-publish"
+
+    id "ru.vyarus.animalsniffer"
+}
+
+description = 'OpenTelemetry Protocol Metrics Exporter'
+ext.moduleName = "io.opentelemetry.exporter.otlp.metrics"
```

Comment:
do we want to put the `-alpha` on the version numbers?

Reply:
Yeah added